src/appsettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appSettings": {
+    "tenantId": "your-tenant.onmicrosoft.com",
+    "appId": "Application (client) ID",
+    "appSecret": "Client secret",
+    "usersFileName": "users.json"
```

Comment:
just an idea: 
we usually set this up like this in our samples:
` "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",`

Can you use ClientId instead of appId? 

`"ClientSecret": "[Copy the client secret added to the app from the Azure portal]"`
`"TenantId": "[Enter 'common', or 'organizations' or the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",`

Reply:
@jennyf19 For the descriptions on that and appSecret, I was going for an exact match with the UI labels in the portal, which are _Application (client) ID_ and _Client secret_, respectively, so am hesitant to modify those.

I'll definitely change `appSecret` to `ClientSecret`, however, and will adjust casing on the others.